clippy_lints/src/lib.rs

```diff
@@ -306,6 +306,8 @@ mod shadow;
 mod single_component_path_imports;
 mod slow_vector_initialization;
 mod stable_sort_primitive;
+mod str_to_string;
+mod string_to_string;
```

Comment:
Those to lints should be in the same module. There is already a `strings` module, so you can put these lints there. 

And while you're there, you could combine all the `LateLintPass` implementations in the `strings` module into one. But please do this in an extra commit. (Or leave it for a later PR)

Reply:
Adding the two lints to the strings.rs file would do it? Also by combining all the `LateLintPass` implementations do you mean that there should be only one of it for all the lints in string.rs file?

Reply:
> do you mean that there should be only one of it for all the lints in string.rs file?

Yes, exactly.

> Adding the two lints to the strings.rs file would do it?

You also have to copy over the code in the `LateLintPass` impls, but you can reuse the `LateLintPass` impl in the other module.

Reply:
I think combining all the `LateLintPass` implementations could be done in a new PR